nemo/collections/asr/metrics/rnnt_wer_bpe.py

```diff
@@ -125,6 +131,7 @@ class RNNTBPEDecoding(AbstractRNNTDecoding):
     def __init__(self, decoding_cfg, decoder, joint, tokenizer: TokenizerSpec):
         blank_id = tokenizer.tokenizer.vocab_size
         self.tokenizer = tokenizer
+        self.compute_langs = decoding_cfg.get('compute_langs', False)
```

Comment:
You can move this line to AbstractRNNTDecoding

Reply:
done